test/js/ez-browse.js

```diff
@@ -0,0 +1,82 @@
+describe('ez-browse', function() {
+    let element;
+
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+    });
+
+    function simulateClick(element) {
+        const click = new CustomEvent('click', {
+            bubbles: true,
+            cancelable: true,
+        });
+
+        element.dispatchEvent(click);
+        return click;
+    }
+
+    it('should be defined', function () {
+        assert.equal(
+            window.customElements.get('ez-browse'),
+            element.constructor
+        );
+    });
+
+    describe('properties', function () {
+        describe('`selectedLocationId`', function () {
+            it('should be set by the attribute', function () {
+                const value = 69;
+
+                element.setAttribute('selected-location-id', value);
```

Comment:
don't set the attribute, it's already set in the fixture, just check that the property is correct